README.md

```diff
@@ -35,6 +35,13 @@ This restriction makes the Prometheus pushgateway inappropriate for the usecase
 
 See https://github.com/weaveworks/promjs/ for a JS client library for Prometheus that can be used from within a web app.
 
+## Getting Started
+Mostly, registering metrics and pushing are the same as the official push gateway
+with a few differences:  
+- There is no job and metrics namespace needed as required by the official one.
```

Comment:
Perhaps because I've never looked at "the official one", I don't know what this means.  Is it possible to phrase it in a positive rather than a negative way?  Or to give some circumstances where the reader will need to act on this advice?